index.html

```diff
@@ -319,6 +321,6 @@ <h3>Skills</h3>
 
       gtag('config', 'G-KWVH6E6BSE');
     </script>
-
+    <script src="/index.js"></script>
```

Comment:
Hello @team, you have worked really hard on this project 💪🏻! However, the menu button does not toggle online. This is because of the forward slash `/index.js` that was added before `index.js`. It may be working locally but the file path was not referenced well. I there fore, kindly suggest that you remove it to ensure that the javascript file is referenced in the website 😄